### PR TITLE
modified the height of the rows for the allocations calendar

### PR DESCRIPTION
**Pull request by Guntrisoft**
- in fullscreen mode on TPAS's monitors this will fit in
  almost a whole day
- ensured that the row highlighter doesn't affect mouse drag

### CODE REVIEW

File: app/assets/javascripts/calendars/company.es6 [hunk 1]
```diff
@@ -20,7 +20,8 @@ class CompanyCalendar extends Calendar {
       },
       groupByDateAndResource: true,
       nowIndicator: true,
-      slotDuration: '00:30:00',
+      slotDuration: '00:10:00',
+      slotLabelInterval: {'minutes': 60},
```
Comment (benbarnett) [inline app/assets/javascripts/calendars/company.es6]: Can we add a space around the braces?

File: app/assets/stylesheets/components/_calendars.scss [hunk 2]
```diff
@@ -24,13 +24,14 @@ $calendar-bookable-slot-background: $color-madang;
 
 .company-calendar {
   .fc-resource-cell {
-    height: 150px;
+    height: 120px;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/_calendars.scss]: Might be good to just make this `120px` a variable `$guider-row-height` or something?

File: app/assets/javascripts/calendars/company.es6 (hunk 1 above)
Comment (Guntrisoft) [inline app/assets/javascripts/calendars/company.es6]: done

File: app/assets/stylesheets/components/_calendars.scss (hunk 2 above)
Comment (Guntrisoft) [inline app/assets/stylesheets/components/_calendars.scss]: done

**Pull request merged**